department-of-veterans-affairs/next-build
# VACMS-16287 additional preview indicator

## Description
Closes [#16287](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16287).


## Testing done
Locally, running `yarn dev` and the cms with ddev.

I enabled the feature flag for next preview, then checked a story node in various moderation states & revisions.


## Screenshots
`Published` workflow state, no banner
![Screenshot 2023-12-11 at 4 31 03 PM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/f271edb7-bda3-4a03-b90e-36ab215259c4)

`Archived` workflow state, says archived revision in banner
![Screenshot 2023-12-11 at 4 29 01 PM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/8b4ab507-7975-466d-a77c-9b28055afb7c)

`Draft` workflow state, says draft revision in banner
![Screenshot 2023-12-11 at 4 32 34 PM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/cab42606-3a09-42b8-9766-e7ac332688d0)

`In Review` workflow state, says draft revision in banner
![Screenshot 2023-12-11 at 4 35 41 PM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/50e0bf58-763f-4dbf-ba01-ed4195d99cbd)



## QA steps
1. In one terminal window, running the CMS locally with ddev start (main branch is fine)
2. In a separate terminal window, in the next-build repo use this branch and start `yarn dev`
3. In the local CMS, enable the FEATURE_NEXT_STORY_PREVIEW feature flag
4. Visit any Story published node, check the preview link. Top banner should not be present.
5. Edit the Story, set the workflow state to `Draft`
6. Check the preview link, notice the top banner is present with `draft revision`
7. Edit the Story again, set the workflow state to `Archived`
8. Check the preview link, notice the top banner has changed to `an archived revision`

```[tasklist]
```

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
